index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head id="header">
+    <link rel="stylesheet" href="styles.css">
+
+</head>
```

Comment:
I kindly suggest having meta tags as a standard template to describe the document. With this meta tag, search engines can easily search your website. The two meta tags I recommend are descriptions and keywords. Besides that, it is a good HTML template to start your HTML/CSS journal 👍